installation/helm-chart/values.yaml.example

```diff
@@ -91,3 +99,13 @@ QUEUE_MAX_TIMEOUT: ""
 SIZE_THRESHOLD: ""
 INSERT_TIME_THRESHOLD: ""
 ACTIVITY_TIME_THRESHOLD: ""
+
+############ RESOURCES LIMITS #############
+API_CPU_LIMIT: "" # "250m"
+API_MEM_LIMIT: "" # "1Gi"
+CELERY_WORKER_CPU_LIMIT: "" # "100m"
+CELERY_WORKER_MEM_LIMIT: "" # "500Mi"
+CELERY_BEAT_CPU_LIMIT: "" # "100m"
+CELERY_BEAT_MEM_LIMIT: "" # "250Mi"
+CONSUMERS_CPU_LIMIT: "" # "250m"
+CONSUMERS_MEM_LIMIT: "" # "750Mi"
```

Comment:
The `CONSUMERS_MEM_LIMIT` and `CONSUMERS_CPU_LIMIT` are not being used.

Reply:
Done.